Datasets/download_s01_data.py

```diff
@@ -5,31 +5,49 @@
 # can be download using pip: pip install wget
 
 import wget
-
+import pandas as pd
+import numpy as np
+import os
 
 # we're downloading only subject_01 data for now, but the url can be altered to download any of the 52 subjects' data
-url = 'ftp://parrot.genomics.cn/gigadb/pub/10.5524/100001_101000/100295/mat_data/s01.mat'
-wget.download(url)
+urls = []
+nums = 53
+for i in range(nums):
+  if i in range(1, 10):
+    number = '{}'.format(i)
+    urls.append('ftp://parrot.genomics.cn/gigadb/pub/10.5524/100001_101000/100295/mat_data/')
+    wget.download(urls[i], out='s'+number.zfill(2)+'.mat')  
+  else:
+    urls.append('ftp://parrot.genomics.cn/gigadb/pub/10.5524/100001_101000/100295/mat_data/')
+    wget.download(urls[i], out='s{}'.format(i)+'.mat')
```

Comment:
- There seems to be an indexing error here. this code downloads 's00' to 's52' but the file names are from 's01' to 's52'.
- Also, `zfill()` method takes care of numbers from 0 to 9 so there's no need to have separate if statement for that.

And rather than downloading all the files, this could be wrapped in a function that takes in a list of subjects that you want to download, and downloads only those subjects' data. Otherwise, all the data is downloaded, even in case we don't need all of them.